reddit_gold/data/tailors.json

```diff
@@ -1,834 +1,734 @@
-                "name": "cake"
+                "name": "gi"
             }, 
             {
-                "name": "downvote"
+                "name": "halter"
             }, 
             {
-                "name": "ds"
+                "name": "heart"
             }, 
             {
-                "name": "wiimote"
+                "name": "historical-costuming"
             }, 
             {
-                "name": "baseball"
+                "name": "imperium"
             }, 
             {
-                "name": "magnifying_glass"
+                "name": "jacket"
             }, 
             {
-                "name": "soccer"
+                "name": "kendo"
             }, 
             {
-                "name": "wrench"
+                "name": "lab-coat"
             }, 
             {
-                "name": "pipe"
+                "name": "military"
             }, 
             {
-                "name": "can"
+                "name": "orangered"
             }, 
             {
-                "name": "rugby_ball"
+                "name": "perwinkle"
             }, 
             {
-                "name": "purple_flower"
+                "name": "rainbow-vest"
             }, 
             {
-                "name": "gilder"
+                "name": "shirt-tie"
             }, 
             {
-                "name": "champagne"
+                "name": "ski-jacket"
             }, 
             {
-                "name": "joystick"
+                "name": "soccer"
             }, 
             {
-                "name": "cat"
+                "name": "star-trek"
             }, 
             {
-                "name": "bowl"
+                "name": "tank"
             }, 
             {
-                "name": "dog"
+                "name": "toga"
             }, 
             {
-                "name": "garden_fork"
+                "name": "tubetop"
             }, 
             {
-                "name": "rabbit"
+                "name": "tux"
             }, 
             {
-                "name": "cupcake"
+                "name": "v-neck-curved"
             }, 
             {
-                "name": "spoon"
+                "name": "v-neck"
             }, 
             {
-                "name": "field_hockey_stick"
+                "name": "vest-warmer"
             }, 
             {
-                "name": "american_football"
+                "name": "vest"
             }, 
             {
-                "name": "baby"
+                "name": "wolverine"
             }, 
             {
-                "name": "basketball"
+                "name": "zip-up"
             }
         ], 
-        "use_dynamic_color": true, 
-        "z-index": 700, 
-        "flippable": true, 
+        "use_dynamic_color": false, 
+        "flippable": false, 
+        "z-index": 300, 
         "flip_x": false, 
-        "ui-order": 4, 
-        "image_path": "grippables"
+        "ui-order": 5
     }
 ]
```

Comment:
I know that this was my own doing, but I think that this file doesn't actually need to be in git if it is getting built from `make`.  should be safe to remove it